library/src/tests/resources/state_preparation.qs

```diff
@@ -49,52 +49,52 @@ namespace Test {
         T(qs[1]);
     }
 
-    operation TestPrepareComplex(): Unit {
+    operation TestPrepareComplex() : Unit {
         use q = Qubit[2];
         let c00 = ComplexPolar(0.5, 0.0);
-        let c01 = ComplexPolar(0.5, PI()/4.0);
-        let c10 = ComplexPolar(0.5, PI()/2.0);
-        let c11 = ComplexPolar(0.5, 3.0*PI()/4.0);
+        let c01 = ComplexPolar(0.5, PI() / 4.0);
+        let c10 = ComplexPolar(0.5, PI() / 2.0);
+        let c11 = ComplexPolar(0.5, 3.0 * PI() / 4.0);
         ApproximatelyPreparePureStateCP(0.0, [c00, c01, c10, c11], q);
         DumpMachine();
         Adjoint PrepareComplex(q);
     }
 
-    operation TestPreparationCompletion(): Unit {
+    operation TestPreparationCompletion() : Unit {
         let testCases = [
-            // Test positive coefficients
-            [0.773761, 0.633478],
+        // Test positive coefficients
+        [0.773761, 0.633478],
             [0.183017, 0.406973, 0.604925, 0.659502],
             [0.0986553, 0.359005, 0.465689, 0.467395, 0.419893, 0.118445, 0.461883, 0.149609],
             [0.271471, 0.0583654, 0.11639, 0.36112, 0.307383, 0.193371, 0.274151, 0.332542, 0.130172, 0.222546, 0.314879, 0.210704, 0.212429, 0.245518, 0.30666, 0.22773],
 
-            // Test negative coefficients; should give same probabilities as positive coefficients
-            [-0.773761, 0.633478],
+        // Test negative coefficients; should give same probabilities as positive coefficients
+        [-0.773761, 0.633478],
```

Comment:
Yup... definitely the comments causing it. Looks like most of these changes in this area are wrong, and always when there's a comment.

Reply:
This, as well as the other places you noticed the indentation is off, is a known limitation of the way the formatter is structured currently. There is no way to fix this without completely rewriting the formatter, unfortunately.

Reply:
Alternatively, I can remove any indentation logic from the formatter, if that is preferrable.

Reply:
Indentation logic will be addressed by a future PR.